control/margins.py

```diff
@@ -340,18 +340,18 @@ def margin(*args):
         Gain margin
     pm : float
         Phase margin (in degrees)
-    Wcg : float
-        Gain crossover frequency (corresponding to phase margin)
-    Wcp : float
-        Phase crossover frequency (corresponding to gain margin) (in rad/sec)
+    wg: float
+        Gain margin crossover frequency (where phase crosses -180 degrees)
```

Comment:
Why not omit "crossover" altogether?  The parenthetical remarks are helpful -- I always get these mixed up.